app/models/post.rb

```diff
@@ -3,6 +3,9 @@ class Post < ApplicationRecord
   has_many :comments
   has_many :likes
 
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
```

Comment:
- You have done well so far, please validate that in your post model, the LikesCounter must be an integer greater than or equal to zero. Please fix this 😄